src/bussinessLogic/ManageMap.java

```diff
@@ -46,6 +46,14 @@
      LetterAttributes la =  (LetterAttributes)map.get("a");
      */
 
+    /**
+     * class constructor
+     */
+
+    public ManageMap() {
+        setNewCharMap();
```

Comment:
The method _setNewCharMap_ can be made private.

I know it is called into two other places in the application, but I think it can be replaced using this approach.

Reply:
So, when I need to create a new map I'll call the constructor of the class (which calls the method)?

Reply:
Yes, this is equivalent with creating a new object.

Reply:
I understand.